hello-world/src/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
code formatting is not right. you can install prettier in your vs code extension. code seems to be fine

Reply:
Thanks for the suggestion. Fixed formatting